13/umbraco-cms/fundamentals/backoffice/property-editors/built-in-umbraco-property-editors/checkbox-list.md

```diff
@@ -102,3 +102,7 @@ If Modelsbuilder is enabled you can get the alias of the desired property withou
 content.SetValue(Home.GetModelPropertyType(_publishedSnapshotAccessor,x => x.SuperHeros).Alias, JsonConvert.SerializeObject(new[] { "Umbraco", "CodeGarden"}));
 }
 ```
+
+## Translating Checkbox List Values Using Dictionary Items
+
+To translate the values of a Checkbox List property editor in a multi-lingual setup, you can use Dictionary items. For more details, see the [Translating Multi-Value Property Editors](../../../../tutorials/multilanguage-setup.md#translating-multi-value-property-editors) section in the [Creating a Multilingual Site](../../../../tutorials/multilanguage-setup.md) article.
```

Comment:
Instead of two links to the same article, I think it's enough using the anchor link with the title of the article 💪 

```suggestion
You can use Dictionary items to translate the values of a Checkbox List property editor in a multilingual setup. For more details, see the [Translating Multi-Value Property Editors](../../../../tutorials/multilanguage-setup.md#translating-multi-value-property-editors) section in the [Creating a Multilingual Site](../../../../tutorials/multilanguage-setup.md) article.
```